src/App.js

```diff
@@ -56,7 +56,6 @@ class App extends Component {
   }
 
   onSettingsRender = () => {
-    return <Settings />
```

Comment:
What does this affect? Do we remove a hole component here?

Reply:
`Settings` does not exist in `src/components/index.js` removing it only prevents the app to crash